src/instructlab/model/evaluate.py

```diff
@@ -333,6 +333,12 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--background",
+    type=click.BOOL,
+    default=False,
```

Comment:
Leaning toward --enable-serving-output as the option and False as the default.  And then something in the help that refers to this option as being helpful for debugging.

Reply:
I need to spend some time and see if llama-cpp's output can be limited in the same way as vLLM's, since everything in this PR only really applies to vLLM and not llama.

Reply:
Why does this have to be a flag when the backend runs behind a main command like chat or eval?

Reply:
This was a request I made because vLLM has useful debug info logging for that isn't caught by ensure_server or by sub-process lifespan.

Reply:
@leseb Though this is inherently a property of serving, we wanted this as a flag for eval for debugging purposes.

Reply:
I'm ok with that @alimaredia